### PR TITLE
Update documentation for squeezebox integration for config flow

### PR DESCRIPTION
**Pull request by rajlaud**
## Proposed change
Update documentation for configuration flow.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [X] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/35669
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/squeezebox.markdown [hunk 1]
```diff
@@ -8,9 +8,11 @@ ha_iot_class: Local Polling
...
 ---
 
-The `squeezebox` platform allows you to control a [Logitech Squeezebox](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29) music player from Home Assistant. This lets you control Squeezebox hardware like the Classic, Transporter, Duet, Boom, Radio and Touch and of software players like [SoftSqueeze](http://softsqueeze.sourceforge.net/), [SqueezePlayer](https://play.google.com/store/apps/details?id=de.bluegaspode.squeezeplayer) and [SqueezeSlave](https://forums.slimdevices.com/showthread.php?93607-ANNOUNCE-Squeezeslave-1-2-released).
+The `squeezebox` platform allows you to control a [Logitech Squeezebox](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29) music player from Home Assistant. This lets you control Squeezebox hardware like the Classic, Transporter, Duet, Boom, Radio and Touch and of software players like [Squeezelite](https://github.com/ralph-irving/squeezelite), [SoftSqueeze](http://softsqueeze.sourceforge.net/), [SqueezePlayer](https://play.google.com/store/apps/details?id=de.bluegaspode.squeezeplayer) and [SqueezeSlave](https://forums.slimdevices.com/showthread.php?93607-ANNOUNCE-Squeezeslave-1-2-released).
```
Comment (frenck) [inline source/_integrations/squeezebox.markdown]:
```suggestion
The Squeezebox integration allows you to control a [Logitech Squeezebox](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29) music player from Home Assistant. This lets you control Squeezebox hardware like the Classic, Transporter, Duet, Boom, Radio and Touch and of software players like [Squeezelite](https://github.com/ralph-irving/squeezelite), [SoftSqueeze](http://softsqueeze.sourceforge.net/), [SqueezePlayer](https://play.google.com/store/apps/details?id=de.bluegaspode.squeezeplayer) and [SqueezeSlave](https://forums.slimdevices.com/showthread.php?93607-ANNOUNCE-Squeezeslave-1-2-released).
```

File: source/_integrations/squeezebox.markdown [hunk 2]
```diff
@@ -8,9 +8,11 @@ ha_iot_class: Local Polling
...
 
-To add your Squeezebox player to your installation, add the following to your `configuration.yaml` file:
+You can add the Squeezebox integration through the frontend configuration menu. This is the preferred method.
```
Comment (frenck) [inline source/_integrations/squeezebox.markdown]: Please describe how to set it up via the frontend. What does a user, who has never worked with Home Assistant, have to do?

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @rajlaud! Left 2 small comments, could you take a look? Thanks! 👍

File: source/_integrations/squeezebox.markdown (hunk 2 above)
Comment (rajlaud) [reply source/_integrations/squeezebox.markdown]: Thanks, this is a helpful way to think about it. I just pushed an updated description. Let me know what you think.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/squeezebox.markdown (hunk 2 above)
Comment (frenck) [reply source/_integrations/squeezebox.markdown]: Awesome!

**Pull request merged**